process/rating/blockSigningRater.go

```diff
@@ -184,15 +186,12 @@ func (bsr *BlockSigningRater) ComputeDecreaseProposer(shardId uint32, currentRat
 		consecutiveBlocksPenalty = bsr.shardRatingsStepHandler.ConsecutiveMissedBlocksPenalty()
 	}
 
-	computedIncreaseBigFloat := big.NewFloat(0).Mul(big.NewFloat(float64(proposerDecreaseRatingStep)),
-		core.Pow(big.NewFloat(float64(consecutiveBlocksPenalty)), uint64(consecutiveMisses)))
-	computedIncrease, _ := computedIncreaseBigFloat.Int64()
-
+	computedFloat := float64(proposerDecreaseRatingStep) * math.Pow(float64(consecutiveBlocksPenalty), float64(consecutiveMisses))
```

Comment:
Maybe we should top up the penalty? Something like 
```GO
if consecutiveMisses >= maxConsecutiveMisses{
         penalty := //a large value
} else {
         penalty := math.Pow(float64(consecutiveBlocksPenalty), float64(consecutiveMisses))
}
```
This is to assure that we do not have the situation in which we might end up with a float overflow.

Reply:
hopefully fixed with a for